docs/semgrep-supply-chain/getting-started.md

```diff
@@ -38,8 +38,11 @@ Semgrep Supply Chain is **automatically enabled** for all repositories that you
 
 When the scan finishes, Semgrep Cloud Platform displays an overview of findings in the **Supply Chain** page for further triage and remediation. See [Triaging and remediating dependency findings](/semgrep-supply-chain/triage-and-remediation).
 
+### Warning on Project Directory Structure
```

Comment:
I don't think this is exactly a warning. I'd probably describe it as something like "Repository structure requirements" (maybe not exactly that, Sara might have an idea).